src/pdp/server.ts

```diff
@@ -119,6 +119,7 @@ export interface PieceAdditionStatusResponse {
 export class PDPServer {
   private readonly _serviceURL: string
   private readonly _authHelper: PDPAuthHelper | null
+  // biome-ignore lint/correctness/noUnusedPrivateClassMembers: TODO: remove this parameter not used
   private readonly _serviceName: string
```

Comment:
oh yeah, this is a remnant of a previous Curio interaction pattern, can just remove this